app/models/organisations_user.js

```diff
@@ -8,5 +9,9 @@ export default Model.extend({
   organisationId:  attr('number'),
   position:        attr('string'),
   user:            belongsTo('user', { async: false }),
-  organisation:    belongsTo('organisation', { async: false })
+  organisation:    belongsTo('organisation', { async: false }),
+
+  isInfoComplete: Ember.computed('position', function(){
+    return this.get('position') && this.get('position').length !== 0 ;
```

Comment:
What if we use `!this.get('position').length` instead of `this.get('position').length !== 0` ? Will that work ?